app/src/main/AndroidManifest.xml

```diff
@@ -718,7 +718,16 @@
             android:name=".component.ui.plan.PDFViewActivity"
             android:label="@string/pdf_viewer_title"
             android:parentActivityName=".component.ui.plan.PlansActivity" />
-        <activity android:name=".component.ui.tufilm.show_ticket"></activity>
+        <activity
+            android:name=".component.ui.tufilm.show_ticket"
```

Comment:
this should not be part of the tufilm, but rather of its own package `ticket`

Reply:
So, you mean we should put all our later activity such as BuyticketActivity, ShowticketActivity and PaymentselectionActivity into package ticket?

Reply:
We plan to insert a new selection named"Event" in the menu of Drawerlayout. It will include not only  kinofilm but also other activity. So I don't know whether it will be better to put all package relating with it such as package ticket,package tufilm into package event?

Reply:
Anything related to ticket sale should be inside the `ticket` package. Please do not move any of the `tufilm` classes.